src/main/java/com/devathon/preguntonic/controllers/GameController.java

```diff
@@ -0,0 +1,72 @@
+package com.devathon.preguntonic.controllers;
+
+import java.util.UUID;
+
+import org.springframework.messaging.handler.annotation.DestinationVariable;
+import org.springframework.messaging.handler.annotation.MessageMapping;
+import org.springframework.messaging.handler.annotation.Payload;
+import org.springframework.messaging.handler.annotation.SendTo;
+import org.springframework.messaging.simp.annotation.SubscribeMapping;
+
+import com.devathon.preguntonic.dto.GameStatusDto;
+import com.devathon.preguntonic.dto.PlayerQuestionResponseDto;
+import com.devathon.preguntonic.dto.QuestionResultDto;
+
+
+@MessageMapping("/rooms/{code}/game")
+public interface GameController {
+    
+    /**
+     * Handle players subscription to the game
+     * @param code Code of the room
+     */
+    @SubscribeMapping
+    void gameSubscription(@DestinationVariable String code);
+
+    /**
+     * Players send when they join the game
+     * @param code Code of the room
+     * @return GameStatusDto with the updated players
+     */
+    @MessageMapping("/players/{playerId}/join")
+    @SendTo("/preguntonic/rooms/{code}/game")
+    public GameStatusDto joinGame(@DestinationVariable String code, @DestinationVariable UUID playerId);
```

Comment:
After test it in docker and connecting with front end, we should add the name of the "Path" variable
```suggestion
    public GameStatusDto joinGame(@DestinationVariable("code") String code, @DestinationVariable("playerId") UUID playerId);
```